lib/interface.ts

```diff
@@ -3,16 +3,22 @@
 import restify = require('restify');
 import bunyan = require('bunyan');
 import BAPI = require('./blpapi-wrapper');
+import Session = require('./apisession/session');
 
 export interface IOurRequest extends restify.Request {
     clientCert?: any;
     blpSession: BAPI.Session;
+    apiSession?: Session;
+}
+
+export interface IBufferedData<T> {
```

Comment:
What's the motivation of putting `IBufferedData` here?  Why not have this be exported by `historical-buffer-manager`?